tools/query_breakdown/src/test/java/com/google/bigquery/LocationTrackerTest.java

```diff
@@ -0,0 +1,38 @@
+package com.google.bigquery;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.IOException;
+import org.junit.Test;
+
+public class LocationTrackerTest {
+
+  @Test
+  public void locationTrackerTestInitializedCorrectly() throws IOException {
+    InputReader ir = new InputReader();
+    String absPath = new File("").getAbsolutePath();
+    ir.readInput(absPath + "/src/test/java/com/google/bigquery"
+            + "/InputTestFiles/multipleLineQuery.txt");
+    assertEquals(4, ir.getLocationTracker().getOriginalPosition(3, 4));
+  }
```

Comment:
Unit tests should be self-contained, i.e. they don't rely on other data. So we shouldn't rely on the file `multipleLineQuery.txt`. Please add a method to `InputReader` such as `readFromString(string input)`.

Reply:
Just added the method in inputreader and changed the tests accordingly